README.md

```diff
@@ -16,24 +16,37 @@ There three categories of metrics that Moneo monitors:
     - Power
     - ECC Counts
     - GPU Throttling
-2.	GPU Profiling Counters
+1.	GPU Profiling Counters
     - SM Activity
     - Memory Dram Activity
     - NVLink Activity
     - PCIE Rate
-3.	InfiniBand Network Counters
+1.	InfiniBand Network Counters
     - IB TX/RX rate
     - IB Port errors
+</details>
+
+<details>
+<summary>Grafana Dashboards</summary>
+
+1. Menu: List of available dashboards.
+![image](https://user-images.githubusercontent.com/70273488/186491552-34d545de-b60d-4cdd-aed4-702e2f34d974.png)
+
+   Note: When viewing GPU dashboards make sure to note whether you are using Nvidia or AMD GPU nodes and select the proper dashboard.
 
-### _Views_
-1. Device Counters
+2. Cluster View: contains  min, max, average across devices for GPU/IB metrics per VM.
+![image](https://user-images.githubusercontent.com/70273488/186491238-ce032f87-cab1-41c4-b32e-9826b8c47b72.png)
+![image](https://user-images.githubusercontent.com/70273488/186491363-3a6cdb8d-0d95-4deb-a6b0-e28711f1ba56.png)
+
+3. GPU Device Counters: Detailed view of node level GPU counters.
 ![image](https://user-images.githubusercontent.com/70273488/173664219-43d8d7b7-a4e6-440a-8373-89ca388ce563.png)
 
-2. Profiling Counters
+4. GPU Profiling Counters: Node level profiling metrics require additional overhead which may affect pworkload performance. Tensor, FP16, FP32, and FP64 activity are disabled by default but can be switched on by CLI command.
```

Comment:
pworkload -> workload

Reply:
ah let me address this

Reply:
> "FP16, FP32, and FP64 activity are disabled by default" are or are not?

They are disabled by default. Does the wording sound ambiguous?

Reply:
Oh sorry, that's my bad. I just confused myself.

Reply:
> pworkload -> workload.

This has been addressed